### PR TITLE
Run dotnet format whitespace

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

**Review by nohwnd — APPROVED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -53,9 +53,6 @@ indent_size = 2
 indent_size = 4
 tab_width = 4
 
-# New line preferences
```
Comment (nohwnd) [inline .editorconfig:56]: Is this a change from default? Please annotate it rather than deleting.

**Review by nohwnd — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (Evangelink) [reply .editorconfig:56]: Looks like my comment was removed when I pushed! I was saying that for some reason this was duplicate with a different value for C#. The entry exists and is set to true for all files at the beginning of the file.

**Review by nohwnd — APPROVED** [review]

**Comment by nohwnd** [comment]
Add it to that blame ignore file once it is in main please.

**Pull request merged**